vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/model/state/VGSFieldState.kt

```diff
@@ -20,6 +21,7 @@ data class VGSFieldState(
         return "isFocusable: $isFocusable\n" +
                 "isRequired: $isRequired\n" +
                 "isValid: $isValid\n" +
+                "errors: $errors\n" +
```

Comment:
can we rename to `validationErrors`, similar to iOS?

Reply:
@dmytrokhl +